advanced-javascript/arrays.spec.js

```diff
@@ -7,11 +7,95 @@ const arrayFunctions = require('./arrays');
 // hint 1. - you need to pass data to the functions and get expected output.
 // hint 2. - you should test the data type being called back, and perform some sort of operation on the data.
 
-describe('Arrays', () => {
-  describe('map', () => {
-    it('should be a function', () => {
-      const map = arrayFunctions.map;
-      expect(typeof map).toBe('object');
+describe("Arrays", () => {
+  // describe('map', () => {
+  //   it('should be a function', () => {
+  //     const map = arrayFunctions.map;
+  //     expect(typeof map).toBe('object');
+  //   });
+  // });
+
+  describe("each", () => {
+    test("maps array of [3,4,5,6] * 2 and returns [6,8,10,12]", () => {
```

Comment:
This is ok, but not really necessary. If the test was to fail it would provide an explanation. Instead, you could write something in plain language like it("iterates through array of numbers and multiplies each by two", ...